ChangeLog

```diff
@@ -1,3 +1,9 @@
+2020-01-21  Johnathan Wong <jowong@bcgsc.ca>
+
+	* Release version 2.1.0
+	* Added support for RNA sequences.
+	* Updated stHashIterator to produce multiple hashes for each seeds.
```

Comment:
```suggestion
	* Updated stHashIterator to produce multiple hashes for each seed
```
Is this the correct meaning? Or `for all seeds`?

Reply:
`Updated stHashIterator with the ability to produce multiple hashes for each seed` Is this better?

Reply:
Also with regards to the period, I see Ben using them in his release notes

Reply:
I don't normally put periods with bullet points, but I don't care too much either way - let's just be consistent (ie. first bullet point didn't have a period)

Reply:
Hmm..
```
stHashIterator is now able to produce multiple hashes for each seed
```
?